### ISSUE TITLE
"Gallu RR3" has wrong ammo type in API

### ISSUE THREAD

**Issue by pancakeslp** [Bungie-net/api#1666]
While looking in DIM, I noticed that "Gallu RR3" is listed as having primary ammo type.
I loaded the sniper rifle in game and then found it uses Special ammo.
So then I looked in light.gg and it agrees with DIM that it is a "primary" https://www.light.gg/db/items/617501162/gallu-rr3/?

So I went to go look in the API via "/common/destiny2_content/json/en/DestinyInventoryItemDefinition-46404b72-f124-491d-95d1-37b9c51a8112.json",
Below is a condensed and formatted snippet

IIRC ammoType 1 is primary, so I think this is a bug in data returned by API

```
    "617501162": {
        "displayProperties": {
            "description": "",
            "name": "Gallu RR3",
            "icon": "/common/destiny2_content/icons/e403e9dab17052a54d453f6f2c87d562.jpg",
            "hasIcon": true
        },
...
        "equippingBlock": {
            "uniqueLabelHash": 0,
            "equipmentSlotTypeHash": 2465295065,
            "attributes": 0,
            "equippingSoundHash": 0,
            "hornSoundHash": 0,
            "ammoType": 1,
            "displayStrings": [
                ""
            ]
        },
```

**Comment by justrealmilk** [comment]
A game side bug! How interesting

**Comment by pancakeslp** [comment]
"Fixed an issue where the Gallu RR3 Sniper Rifle claimed that it used Primary ammo. "
https://www.bungie.net/en/News/Article/51593

Closing as this was resolved in UPDATE 4.1.5
